jbmc/src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -1038,6 +1038,26 @@ bool java_bytecode_languaget::convert_single_method(
     return false;
   }
 
+  const auto type_adder = [&needed_lazy_methods](const typet &type) {
+    const pointer_typet *return_type_pointer =
+      type_try_dynamic_cast<pointer_typet>(type);
+    if(return_type_pointer)
+    {
+      const symbol_typet *symbol_type =
+        type_try_dynamic_cast<symbol_typet>(return_type_pointer->subtype());
+      if(symbol_type)
+      {
+        needed_lazy_methods->add_needed_class(symbol_type->get_identifier());
+      }
+    }
+  };
+
+  // opaque method, but we should load the classes of the parameters and return??
```

Comment:
Incomplete comment?

Reply:
I'll squash some commits so this goes away.